src/evrard/evrard_new.cpp

```diff
@@ -42,13 +47,13 @@ int main(int argc, char** argv)
         return exitSuccess();
     }
 
-    const size_t maxStep = parser.getInt("-s", 10);
     const size_t nParticles = parser.getInt("-n", 65536);
+    const size_t maxStep = parser.getInt("-s", 10);
     const int writeFrequency = parser.getInt("-w", -1);
     const int checkpointFrequency = parser.getInt("-c", -1);
     const bool quiet = parser.exists("--quiet");
     const std::string checkpointInput = parser.getString("--cinput");
-    const std::string inputFilePath = parser.getString("--input", "bigfiles/Test3DEvrardRel.bin");
+    const std::string inputFilePath = parser.getString("--input", "../../../src/evrard/bigfiles/Test3DEvrardRel.bin");
```

Comment:
we could change this to
```suggestion
    const std::string inputFilePath = parser.getString("--input", "./Test3DEvrardRel.bin");
```